lib/crowi/express-init.js

```diff
@@ -12,10 +12,27 @@ module.exports = function(crowi, app) {
     , flash          = require('connect-flash')
     , cons           = require('consolidate')
     , swig           = require('swig')
+    , i18next        = require('i18next')
+    , i18nFsBackend  = require('i18next-node-fs-backend')
+    , i18nSprintf    = require('i18next-sprintf-postprocessor')
+    , i18nMiddleware = require('i18next-express-middleware')
     , env            = crowi.node_env
     , middleware     = require('../util/middlewares')
     ;
 
+  i18next
+    .use(i18nMiddleware.LanguageDetector)
```

Comment:
Are you sure how `i18nMiddleware.LanguageDetector` detect user language?
I'm using Mac and browser with English mode and found that the UI was still Japanese though (on [review app](https://crowi-dev-pr-143.herokuapp.com/)).